packages/react/src/lib/positioner/types.ts

```diff
@@ -66,7 +67,7 @@ export interface PopperProps extends PositioningProps {
   /**
    * Ref object containing the target node (the element that we're using as reference for Popper box).
    */
-  targetRef?: React.RefObject<Element>
+  targetRef: React.RefObject<Element>
```

Comment:
`Popper` won't work without providing this, I think it makes sense to mark it as required